src/test/java/org/broadinstitute/consent/http/models/DataAccessRequestDataTest.java

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.consent.http.models;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.junit.jupiter.api.Test;
+
+class DataAccessRequestDataTest {
+
+  @Test
+  void serialization() {
+    String exampleDarData = """
+        {
+          "institution": "The Broad Institute of MIT and Harvard",
+          "projectTitle": "title",
+          "checkCollaborator": false,
+          "checkNihDataOnly": false
+        }""";
+    DataAccessRequestData resultingDarData = DataAccessRequestData.fromString(exampleDarData);
+    String expectedDarData = """
+        {"projectTitle":"title","checkNihDataOnly":false}""";
```

Comment:
Is the order of properties deterministic? If not, this may fail on different systems.

The [JSONAssert](https://github.com/skyscreamer/JSONassert) library is a good way to test JSON objects, but consent doesn't use it yet. It's bundled with spring-boot so It is used by other terra services for testing.

Reply:
I'm pretty sure Jackson can do this too. Not sure which is better, but if we have the capability already, I'd prefer to use what we have.